src/rpc/stun.js

```diff
@@ -538,6 +538,7 @@ function encode_attr_xor_mapped_addr(addr, buffer, offset, end) {
  * encode ERROR-CODE attribute
  */
 function encode_attr_error_code(err, buffer, start, end) {
+    // eslint-disable-next-line no-bitwise
```

Comment:
this file is the only one that disables this lint rule for the entire file because it heavily uses bitwise operators, so this specific line is redundant.

Reply:
Not what the linter said. It said that I should add each line since they are for the next-line only.

Reply:
I meant that the stun.js file already has that "whole file disable rule" for no-bitwise...
see here - https://github.com/noobaa/noobaa-core/blob/master/src/rpc/stun.js#L2
So you didn't need to add that in this case :)